charts/gardener/controlplane/charts/runtime/templates/apiserver/deployment.yaml

```diff
@@ -275,6 +278,9 @@ spec:
         {{- end }}
         - --workload-identity-token-min-expiration={{ .Values.global.apiserver.workloadIdentity.token.minExpiration }}
         - --workload-identity-token-max-expiration={{ .Values.global.apiserver.workloadIdentity.token.maxExpiration }}
+        {{- if or .Values.global.apiserver.workloadIdentity.token.signing.secretName .Values.global.apiserver.workloadIdentity.token.signing.key }}
+        - --workload-identity-signing-key-file=/etc/gardener-apiserver/workload-identity/tls/tls.key
```

Comment:
```suggestion
        - --workload-identity-signing-key-file=/etc/gardener-apiserver/workload-identity/signing.key
```

Reply:
The full filepath is now `/etc/gardener-apiserver/workload-identity/signing/key.pem`.